packages/core/src/common/browserSupport.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+export function elementClosest(element: Element, selector: string): Element | null {
```

Comment:
if we do remove the function below then i would support putting this in `utils` directly, as it's actually quite a useful DOM utility.